tools/data_usage/console.cpp

```diff
@@ -1,109 +1,139 @@
 // Copyright (c) Microsoft Corporation
 // SPDX-License-Identifier: MIT
 
-#include <windows.h>
 #include <iostream>
 #include <string>
+#include <windows.h>
+#include "bpf.h"
 #include "ebpf_api.h"
+#include "libbpf.h"
 #include "ebpf_nethooks.h"
 
-#define MAP_COUNT 1
+typedef struct _app_name
+{
+    wchar_t name[32];
+} app_name_t;
 
 int main(int argc, const char** argv)
 {
     UNREFERENCED_PARAMETER(argc);
     UNREFERENCED_PARAMETER(argv);
 
-    ebpf_handle_t flow_program_handle;
-    ebpf_handle_t flow_map_handle[MAP_COUNT];
-    ebpf_handle_t flow_link_handle;
-
-    ebpf_handle_t mac_program_handle;
-    ebpf_handle_t mac_map_handle[MAP_COUNT];
-    ebpf_handle_t mac_link_handle;
+    const char* error_message = NULL;
+    ebpf_result_t result;
 
-    uint32_t count_of_map_handle = MAP_COUNT;
-    const char* error_message = nullptr;
-
-    uint32_t result = ebpf_api_initiate();
-    if (result != ERROR_SUCCESS)
+    uint32_t initiate_result = ebpf_api_initiate();
+    if (initiate_result != ERROR_SUCCESS)
     {
-        std::cerr << "ebpf_api_initiate failed: " << result << std::endl;
+        fprintf(stderr, "ebpf_api_initiate failed: %d\n", initiate_result);
```

Comment:
%d is for signed integers. This type and others are unsigned so you should use %u or %lu.